internalsdk/pro/responses.go

```diff
@@ -0,0 +1,37 @@
+package pro
```

Comment:
@atavism , Do you think, we should start using protoc file from lantern-cloud?, I think that would be much better if we didn't have to maintain multiple files. Let me know what do you think?

Reply:
FYI- we can do later also, But I think doing that we can save bit of time and we don't have keep maintain same file as server.

Reply:
> Do you think, we should start using protoc file from lantern-cloud?, I think that would be much better if we didn't have to maintain multiple files.

Yeah, I agree it would be much better not having to maintain multiple proto files, but I don't think we can have lantern-cloud as a dependency here. One reason is because it isn't open-source and this project is now. Maybe the best solution is to move the proto files to a repo like https://github.com/getlantern/common instead and update lantern-cloud/lantern-client to use it?

Reply:
@atavism Can't we add this [apipb](https://github.com/getlantern/lantern-cloud/blob/7372fa0b96888e2cf5cb5ae1f775a2b136850720/cmd/api/apipb/users.proto#L2) package and use it?